config/dockerfiles/Dockerfile.Rust

```diff
@@ -0,0 +1,27 @@
+FROM debian:jessie
```

Comment:
Like I said, my preference is to not handle anything other than officially supported images (I realize I said "community image" above; sorry for any confusion. I mean official. My bad), unless there's an explicit need from the stakeholder(s). But stuff like this is @bokmann's decision, so I'm going to let him weigh in.